include/GafferSceneUI/TransformTool.h

```diff
@@ -255,6 +259,7 @@ class GAFFERSCENEUI_API TransformTool : public GafferSceneUI::SelectionTool
 		bool m_handlesDirty;
 
 		mutable std::vector<Selection> m_selection;
+		mutable bool m_selectionEditable;
```

Comment:
What precipitates the need for the const juggling here, do we have to make `selectionEditable` const or is this related to the above that its just a cache for efficiency?

Reply:
Same as for `m_selection`, we need to compute it lazily, so the accessor method doesn't mutate it conceptually as far as outsiders are concerned.